src/tess_asteroids/movingtpf.py

```diff
@@ -2611,7 +2615,7 @@ def _make_tpf_hdulist(self):
                 name="RAW_CNTS",
                 format=tform.replace("E", "I"),
                 dim=dims,
-                unit="e-/s",
+                unit="ADU",
```

Comment:
why are we changing from -e/s to ADU? If the image data comes from calibrated FFIs, then the units should be -e/s